### ISSUE TITLE
Kalu'ak Fishing Derby Early Start

### ISSUE THREAD

**Issue by gtsmald0ne** [dalaranwow/dalaran-wow#5743]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I caught a blacktip shark before the announcement of the start of the fishing derby, and was able to turn it in after the announcement of the start but only for the 2nd lace prize.

**Expected behaviour**: The derby should have started at 2p, only allowing for the catch of blacktip sharks after that time.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Dragongl11** [comment]
Same here - caught Blacktip Shark like few minutes before event have been started, and even when there was announce, i turn in quest and got second price. Need to fix...

**Comment by galacticat** [comment]
This is one of my favorite events. Would be really nice to see this fixed.
